src/battle_anim.c

```diff
@@ -182,6 +178,94 @@ static void (* const sScriptCmdTable[])(void) =
     Cmd_createdragondartsprite,      // 0x34
 };
 
+static const u16 sMovesWithQuietBGM[] =
+{
+    MOVE_SING, MOVE_PERISH_SONG, MOVE_GRASS_WHISTLE
+};
+
+static const u8* const sBattleAnims_StatusConditions[NUM_B_ANIMS_STATUS] =
+{
+    [B_ANIM_STATUS_PSN]         = Status_Poison,
+    [B_ANIM_STATUS_CONFUSION]   = Status_Confusion,
+    [B_ANIM_STATUS_BRN]         = Status_Burn,
+    [B_ANIM_STATUS_INFATUATION] = Status_Infatuation,
+    [B_ANIM_STATUS_SLP]         = Status_Sleep,
+    [B_ANIM_STATUS_PRZ]         = Status_Paralysis,
+    [B_ANIM_STATUS_FRZ]         = Status_Freeze,
+    [B_ANIM_STATUS_CURSED]      = Status_Curse,
+    [B_ANIM_STATUS_NIGHTMARE]   = Status_Nightmare,
+    [B_ANIM_STATUS_WRAPPED]     = Status_Powder,
```

Comment:
where does the wrapped anim come from?

Reply:
Hm, not sure. According to the comment it doesn't seem to have any use, and `Status_Powder` is dummied anyway. Guess it can be removed.